src/auto/text_buffer.rs

```diff
@@ -274,9 +284,13 @@ impl TextBuffer {
         }
     }
 
-    //pub fn get_serialize_formats(&self) -> (/*Ignored*/Vec<gdk::Atom>, i32) {
-    //    unsafe { TODO: call ffi::gtk_text_buffer_get_serialize_formats() }
-    //}
+    pub fn get_serialize_formats(&self) -> (Vec<gdk::Atom>, i32) {
```

Comment:
Updated test. Seems need use `from_glib_container_num` instead `from_glib_container`. Not checked this.